node-manager/src/error.rs

```diff
@@ -73,6 +75,12 @@ pub enum MutinyStorageError {
     Other(#[from] anyhow::Error),
 }
 
+impl MutinyError {
```

Comment:
I feel like this impl shouldn't be necessary but I guess this is because there the extra layer of indirection? io::Error -> MutinyStorageError -> MutinyError